go/oasis-test-runner/scenario/e2e/stake_cli.go

```diff
@@ -333,6 +338,28 @@ func (s *stakeCLIImpl) testAmendCommissionSchedule(childEnv *env.Env, cli *cli.H
 	return nil
 }
 
+func (s *stakeCLIImpl) getInfo(childEnv *env.Env) error {
+	s.logger.Info("querying common token info")
+	args := []string{
+		"stake", "info",
+		"--" + grpc.CfgAddress, "unix:" + s.basicImpl.net.Validators()[0].SocketPath(),
+	}
+
+	b, err := cli.RunSubCommandWithOutput(childEnv, s.logger, "info", s.basicImpl.net.Config().NodeBinary, args)
+	if err != nil {
+		return fmt.Errorf("scenario/e2e/stake: failed to query common token info: %s error: %w", b.String(), err)
+	}
+	// Check that subcommand reported warnings for invalid staking threshold kinds.
+	subCmdOutput := b.String()
+	thresholdKinds := []string{"compute", "storage"}
+	for _, kind := range thresholdKinds {
+		if !strings.Contains(subCmdOutput, fmt.Sprintf("invalid staking threshold kind: %s", kind)) {
+			return fmt.Errorf("scenario/e2e/stake: querying common token info should warn about invalid staking threshold for kind: %s", kind)
+		}
+	}
```

Comment:
I didn't know if this check is something we would want or is it testing a detail that is not important enough to be covered?